### ISSUE TITLE
Benchmark Bohm/gyro-Bohm model

### ISSUE THREAD

**Issue by theo-brown** [google-deepmind/torax#452]
Prompted by #446, it would be good to have a benchmark of the BgB model as implemented in TORAX vs other codes.

**Comment by theo-brown** [comment]
I think it might be more standard to set `v_face_el ∝d_face_el`. The term I previously implemented for  `v_face_el` should be an additional term, rather than the only term.
Set up a draft branch for this https://github.com/google-deepmind/torax/tree/bgb-convectivity.
Will check that nothing breaks in the tests and then turn it into a PR.

**Comment by jcitrin** [comment]
Hi @theo-brown . With this one in https://github.com/google-deepmind/torax/pull/841 it might be easier to compare now in a simple benchmark with JETTO?

**Comment by theo-brown** [comment]
Yes, can do! I'm not sure what would be best to use as a test case. Ideally we'd run it on a JET simulation, I suppose?

**Comment by jcitrin** [comment]
JET, or STEP? Doesn't really matter as long as JETTO + TORAX can have the same initial conditions, models, etc.

It might be a bit easier after the IMAS I/O is done.

**Comment by jcitrin** [comment]
Closing since redundant with ongoing task to validate against STEP simulations, which will include a plethora of physics-model comparisons
